Server/tools/cad/modeling.py

```
@@ -0,0 +1,209 @@
+    Also zuerst Sketch 1 dann Sketch 2 dann Sketch 3 usw.
+    """
+    try:
+        endpoint = get_endpoints("cad")["loft"]
+        headers = get_headers()
+        data = {
+            "sketchcount": sketchcount
+        }
+        return send_request(endpoint, data, headers)
+
+    except requests.RequestException as e:
+        logging.error("Loft failed: %s", e)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)